bridgetown-website/frontend/javascript/index.js.rb

```diff
@@ -6,10 +6,10 @@
 import '@shoelace-style/shoelace/dist/components/button/button.js'
 import '@shoelace-style/shoelace/dist/components/card/card.js'
 import '@shoelace-style/shoelace/dist/components/dialog/dialog.js'
-import '@shoelace-style/shoelace/dist/components/form/form.js'
```

Comment:
It's missing after beta.63

Reply:
Ah, well we can remove that actually, I don't think we have a form on the site.

Reply:
It's missing because it was deprecated! Shoelace form-associated elements work with regular <form> now